#base_wheel_model:   #底盘轮胎参数（双轮差速）
ratio: 1000.0      #转速转换比例，执行速度调整比例
D: 0.44       #两轮间距，单位是m
S: 0.2             #轮子直径，单位是m
serial_model: 1    #串口模式 0 1:232模式 2 3





